include/cuco/hash_functions.cuh

```diff
@@ -16,10 +16,35 @@
 
 #pragma once
 
-#include <cuco/detail/probe_sequence_impl.cuh>
+#include <cuco/detail/hash_functions/murmurhash3.cuh>
+#include <cuco/detail/hash_functions/xxhash.cuh>
+
+#include <cuda/std/type_traits>
```

Comment:
```suggestion
```
seems irrelevant

Reply:
Good catch! That's a leftover from when I experimented with the `default_hash_function` alias.